qucs/qucs/mouseactions.cpp

```diff
@@ -319,6 +356,8 @@ void MouseActions::MMoveElement(Schematic *Doc, QMouseEvent *Event)
  */
 void MouseActions::MMoveWire2(Schematic *Doc, QMouseEvent *Event)
 {
+//ALYS - i saved old code inside #if 0 block
```

Comment:
please don't do that. we use git.
this way makes review harder, due to lack of an meaningful diff.